### ISSUE TITLE
[1.16.5 / 1.1.1 BUG] Nether biomes from BYG, BoP, and Cinderscapes reading as "minecraft:ocean" in F3 menu after relog

### ISSUE THREAD

**Issue by ZilverLazarus** [Someone-Else-Was-Taken/BetterNether-Reforged#38]
MC Version: 1.16.5
Mod Version: 1.1.1
Forge Version: 36.2.4

Issue(s): After creating a world, going into the nether, exiting the world, and rejoining the world, any nether biomes generated in new chunks that don't belong to Better Nether Reforged are designated as "minecraft:ocean". It also appears that /locatebiome breaks upon leaving and rejoining the world again. Trying to locate the biome that's supposed to be present or "minecraft:ocean" yields no result.

Evidence:
![2021-10-30_21 46 54](https://user-images.githubusercontent.com/83998931/139563332-27f58a4c-6f3e-41b8-bfad-91a4a5c9eb67.png)
![2021-10-30_21 47 00](https://user-images.githubusercontent.com/83998931/139563350-535f6894-734b-4a4c-91af-2d1ca672d95d.png)
![2021-10-30_21 55 58](https://user-images.githubusercontent.com/83998931/139563488-4042cda0-1503-403f-8e37-b926ef459efc.png)

**Comment by Someone-Else-Was-Taken** [comment]
I'll have to look into this issue some more.
Do new non BN/Vanilla biomes still generate?

**Comment by ZilverLazarus** [comment]
> I'll have to look into this issue some more. Do new non BN/Vanilla biomes still generate?

They still generate, but are marked as "minecraft:ocean" on F3. **Screenshot from world "Debug Nether 6" after relog**
![2021-11-02_23 05 33](https://user-images.githubusercontent.com/83998931/140004873-9704ecde-2bcd-4446-90ee-1832b0c16882.png)

I also noticed something else about this issue that's noteworthy. It seems that leaving another world save, immediately creating a new one without closing the game, joining the world and going into the nether causes this issue. The issue persists between worlds. **Screenshot from world "Debug Nether 5"**
![2021-11-02_22 51 39](https://user-images.githubusercontent.com/83998931/140005102-811cdd08-04ca-4b45-99b4-2aa03a85e591.png)

On the flipside... Closing the game, relaunching, creating a new world save and going into the nether doesn't present this issue initially. **Screenshot from world "Debug Nether 6" before relog**
![2021-11-02_23 01 11](https://user-images.githubusercontent.com/83998931/140005314-01068ca7-38c7-41ae-8e6f-51fb90a69660.png)
